pkg/api/login_oauth.go

```diff
@@ -68,6 +68,26 @@ func (hs *HTTPServer) OAuthLogin(ctx *models.ReqContext) {
 		return
 	}
 
+	// Get the URL of the current request
+	scheme := ctx.Req.Header.Get("X-Forwarded-Proto")
+	if scheme == "" {
+		scheme = string(hs.Cfg.Protocol)
+	}
+
+	path := ""
+	uri, err := url.ParseRequestURI(ctx.Req.RequestURI)
+	if err != nil {
+		path = "login/" + name
+	} else {
+		path = uri.Path
+	}
+
+	requestURL := url.URL{
+		Scheme: scheme,
+		Host:   ctx.Req.Host,
+		Path:   path,
```

Comment:
Could you please adjust it in the following way:
```suggestion
		Path:   "/graph" + path,
```